### PR TITLE
[KED-2424] Extend prompts config

### PR DESCRIPTION
**Pull request by andrii-ivaniuk**
https://jira.quantumblack.com/browse/KED-2424

### CODE REVIEW

File: astro-iris/prompts.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ repo_name:
     Please enter a directory name for your new project repository.
     Alphanumeric characters, hyphens and underscores are allowed.
     Lowercase is recommended.
+  regexp_validator: "^\\w+(-*\\w+)*$"
+  error_msg: |
```
Comment (lorenabalan) [inline astro-iris/prompts.yml:15]: When does this error message appear? Is all validation done at once, then spits out all the error, or it checks each input in turn, then immediately spit out error if one is incorrect?

File: spaceflights/prompts.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+project_name:
+  title: "Project Name:"
+  text: |
+    Please enter a human readable name for your new project.
+    Spaces and punctuation are allowed.
+  default: New Kedro Project
```
Comment (lorenabalan) [inline spaceflights/prompts.yml:6]: How are defaults handled for the prompts below? I know atm the default does not appear in the prompts (`[new_kedro_project]: `), is that something we can change as part of this change or not really?

**Review by lorenabalan — APPROVED** [review]
Looks good in principle just had a couple of questions.

**Comment by lorenabalan** [comment]
I like the name `prompts.yml` as well. ✅

File: astro-iris/prompts.yml (hunk 1 above)
Comment (andrii-ivaniuk) [reply astro-iris/prompts.yml:15]: It checks each input in turn and if it is incorrect it emits an error.

File: spaceflights/prompts.yml (hunk 2 above)
Comment (andrii-ivaniuk) [reply spaceflights/prompts.yml:6]: At the moment a default appears in the prompt if it is specified in `prompts.yml` but currently we do not support the behavior when `project_name` was used as a default for other inputs. Now you need to type all the inputs (if no defaults).

File: astro-iris/prompts.yml (hunk 1 above)
Comment (andrii-ivaniuk) [reply astro-iris/prompts.yml:15]: So, the user will be prompted until he/she enters the correct value (as we had before) :)

**Pull request merged**